pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -141,6 +144,39 @@ var _ = Describe("nic hotplug on virt-launcher", func() {
 		),
 	)
 
+	var _ = Describe("set interface state up/down", func() {
+		DescribeTable("interface in current and desired domains",
+			func(domainFrom *api.Domain, domainTo *api.Domain, result libvirtClientResult) {
+				networkInterfaceManager := newVirtIOInterfaceManager(
+					mockLibvirtClient(gomock.NewController(GinkgoT()), result),
```

Comment:
I'm not happy to see this mock, but it seems to be a systematic problem in this package.
We need to get rid of it (in a follow up) so I wont waste my time interpreting what is going on here.